nodeList.json

```diff
@@ -0,0 +1,895 @@
+[
+   {
+      "name":"Willowdale Mesh Node",
+      "latitude":43.77031350236652,
+      "longitude":-79.41487752247622,
+      "onlineStatus":"Online",
+      "cardinalDirection":"Omni",
+      "floor":"0",
+      "status":"active",
+      "dateAdded":"2017-15-18T00:00:00.0Z"
```

Comment:
I think the new commit still carried forward an issue with the date
`2017-15-18T00:00:00.0Z`
should be `2017-08-15T00:00:00.0Z`